webapp/components/DeleteData/DeleteData.vue

```diff
@@ -8,7 +8,9 @@
       {{ $t('settings.deleteUserAccount.pleaseConfirm', { confirm: currentUser.name }) }}
     </label>
     <ds-input v-model="enableDeletionValue" />
-    <p class="notice">{{ $t('settings.deleteUserAccount.accountDescription') }}</p>
+    <p v-show="enableDeletionValue" class="notice">
+      {{ $t('settings.deleteUserAccount.accountDescription') }}
+    </p>
     <label class="checkbox">
       <input
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-26T15:03:19Z" title="Wednesday, August 26th 2020, 5:03:19 pm +02:00">Aug 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -8,8 +8,10 @@
       {{ $t('settings.deleteUserAccount.pleaseConfirm', { confirm: currentUser.name }) }}
     </label>
     <ds-input v-model="enableDeletionValue" />
-    <p class="notice">{{ $t('settings.deleteUserAccount.accountDescription') }}</p>
-    <label class="checkbox">
+    <p v-show="enableDeletionValue" class="notice">
+      {{ $t('settings.deleteUserAccount.accountDescription') }}
+    </p>
+    <label v-if="currentUserCounts.contributionsCount" class="checkbox">
```


```suggestion
    <label class="checkbox">
```

Here comes the error back you fixed in your merged PR for the missing checkbox. 😬🤔